### PR TITLE
update website/versioned_docs/version-5.x/create-single-spa.md

### PR DESCRIPTION
**Pull request by JeffreyLewis-cd**
I pull the latest version single-spa.js.org and update some basic config as instruction. I run this project successfully on my computer and plan to translate some documents to simplified Chinese.

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ module.exports = {
         alt: 'single-spa Logo',
         src: 'img/logo-white-bgblue.svg',
       },
-      links: [
+      items: [
```
Comment (joeldenning) [inline website/docusaurus.config.js:105]: Is this required to upgrade to `@docusaurus/core@^2.0.0-alpha.70`?

**Review by joeldenning — COMMENTED** [review]
Hi @JeffreyLewis-cd, for upgrading dependencies it is best to update the upstream repository first, which will cause this repository to change.

Could you make a pull request to https://github.com/single-spa/single-spa.js.org?

**Comment by joeldenning** [comment]
Hi @JeffreyLewis-cd -

**Comment by JeffreyLewis-cd** [comment]
I met some issue when run this project on my computer. I checked the error information and fixed it. 

https://github.com/single-spa/zh-hans.single-spa.js.org/issues/156

**Pull request closed without merging**